docs/compliance/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+compliance-trestle = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.9"
```

Comment:
I think that we are using 3.10 in our app, can we use the same major version here for consistency?